sdk/state/open_test.go

```diff
@@ -29,14 +31,136 @@ func TestProposePayment_validAsset(t *testing.T) {
 	})
 
 	native := NativeAsset{}
-	_, err := sendingChannel.ProposeOpen(OpenParams{Asset: native})
+	_, err := sendingChannel.ProposeOpen(OpenParams{
+		Trustlines: []Trustline{Trustline{
+			Asset: native},
+		},
+	})
```

Comment:
Can we leave native out of the list, since it should be available by default? Or is that happening in a follow up PR?

Reply:
ya  technically you're right it's not needed here. There's a couple places where I left NativeAsset as a place holder since this ticket isn't finished yet (doing the close steps next). And will remove those in that PR.

I added a TODO here to remind me: https://github.com/stellar/experimental-payment-channels/pull/139/commits/7225ca86fb88774497d0dc1134fdcce851627dd2